interfaces/builtin/lvm_control.go

```diff
@@ -0,0 +1,78 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const lvmControlSummary = `allows control of LVM devices`
+
+const lvmControlBaseDeclarationPlugs = `
+  lvm-control:
+    allow-installation: false
+    deny-auto-connection: true
+`
+
+const lvmControlBaseDeclarationSlots = `
+  lvm-control:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const lvmControlConnectedPlugAppArmor = `
+# Description: Allow control of LVM configuration and sockets.
+
+@{PROC}/devices r,
+/run/udev/data/b[0-9]*:[0-9]* r,
+/sys/block/ r,
+/sys/devices/**/block/** r,
+
+/dev/mapper/control rw,
```

Comment:
This gives ability to control more targets than just LVM.

Reply:
Indeed it does, see below.